src/SignalR/server/Core/src/Internal/HubFilterFactory.cs

```diff
@@ -90,7 +90,7 @@ private ValueTask DisposeFilter(IHubFilter filter)
             var filter = (IHubFilter?)serviceProvider.GetService(_filterType);
             if (filter == null)
             {
-                filter = (IHubFilter)_objectFactory.Invoke(serviceProvider, null);
+                filter = (IHubFilter)_objectFactory.Invoke(serviceProvider, Array.Empty<object>());
```

Comment:
@pranavkm Should the ObjectFactory allow null arguments?

Reply:
Yeah, it should. Could you file an issue? There's more things in the ObjectFactory that should be nullable that I missed.